compiler/rustc_feature/src/active.rs

```diff
@@ -384,6 +384,9 @@ declare_features! (
     (active, exclusive_range_pattern, "1.11.0", Some(37854), None),
     /// Allows exhaustive pattern matching on types that contain uninhabited types.
     (active, exhaustive_patterns, "1.13.0", Some(51085), None),
+    /// Allows using `efiapi`, `sysv64` and `win64` as calling convention
+    /// for functions with varargs.
+    (active, extended_varargs_abi_support, "1.64.0", Some(100189), None),
```

Comment:
I think we've missed the 1.64 cutoff. Can you change this to 1.65?

Reply:
I believe what was meant was to change 

```suggestion
    (active, extended_varargs_abi_support, "1.65.0", Some(100189), None),
```